Chap_Terms.tex

```diff
@@ -120,8 +128,8 @@ \section{Naming Conventions}
 The \ac{PMIx} standard has adopted the following conventions:
 
 \begin{itemize}
-\item \ac{PMIx} constants and attributes are prefixed with \textbf{\code{PMIX}}.
-\item Structures and type definitions are prefixed with \code{pmix}.
+\item \ac{PMIx} constants and attributes are prefixed with \textbf{\code{PMIX_}}.
+\item Structures and type definitions are prefixed with \code{pmix_}.
 \item Underscores are used to separate words in a function or variable name.
```

Comment:
Not sure we discussed this in our latest meetings... do we want this change?

Reply:
I think we do want this change.